config.example.wikimedia.yaml

```diff
@@ -36,15 +36,21 @@ templates:
       license:
         name: Apache2
         url: http://www.apache.org/licenses/LICENSE-2.0
-    security:
-      # ACLs for public *.wikipedia.org wikis
-      - mediaWikiAuth:
-        - user:read
     x-subspecs:
       - mediawiki/v1/content
       - mediawiki_v1_graphoid
       - mediawiki/v1/mobileapps
     # - mediawiki/v1/revision-scoring
+    securityDefinitions:
+      mediawiki_auth:
+        description: Checks permissions using MW api
```

Comment:
This description should ideally be useful to third-party developers. Perhaps something like "MediaWiki cookie authentication", so that they get an idea what kind of cookie they are supposed to pass in here?